src/FSharpPlus/Extensions/Seq.fs

```diff
@@ -20,6 +20,17 @@ module Seq =
     /// <exception cref="System.ArgumentNullException">Thrown when the input sequence is null.</exception>
     let bind (mapping: 'T->seq<'U>) source = Seq.collect mapping source
 
+    /// <summary>Applies a sequence of functions to a sequence of values and concatenates them</summary>
+    /// <param name="f">The seq of functions.</param>
+    /// <param name="x">The seq of values.</param>
+    /// <returns>A seq concatenating the results from applying each function to each value</returns>
+    /// 
+    /// <example>
+    /// <code>
+    /// > Seq.apply [double; triple] [1; 2; 3];;  
+    /// val it : seq<int> = seq [2; 4; 6; 3; ...]
```

Comment:
You probably need to xml-escape the `<int>`

Reply:
Yes, it was that. The generic brackets. At least in my case.

Reply:
Thanks!